scripts/e2e

```diff
@@ -3,6 +3,7 @@ set -e
 
 cd $(dirname $0)
 ./test
+#./validate
```

Comment:
Why was this commented out - should we just remove the line?

Reply:
Yes we should remove it.